data/journeys/docsite-journeys.md

```diff
@@ -31,90 +31,113 @@ Milestone: Scale
 
 ## Novice
 
+Get started with Ansible box content.
```

Comment:
```suggestion
```

Reply:
I think the pattern should be that we build the docsite on the journeys and not the other way around.

Reply:
@oraNod what do you mean? This PR is an attempt to reflect the work we did. We already agreed that, in future, we'll do changes to the journeys first, then we'll reflect them in the templates, etc.
If you think my changes are inconsistent, please feel free to commit any changes right here.

Reply:
@Andersson007 Hey, apologies if I wasn't clear. I meant that adding this line seems to make the journeys correspond to elements on the docsite. i.e. "Get started with Ansible box content." refers to a specific section on the docsite, right?

Reply:
@oraNod yep (as folks who will add new stuff in future can be confused a bit without this comment i believe)

Reply:
@Andersson007 right maybe we should just be more explicit about the sign posting.